sql/2022/fonts/font_writing_scripts.sql

```diff
@@ -0,0 +1,100 @@
+CREATE TEMP FUNCTION detect(codepoints ARRAY<STRING>)
+RETURNS ARRAY<STRING>
+LANGUAGE js AS r"""
+
+# This function and data taken from https://github.com/bramstein/detect-writing-script (BSD-3-Clause)
```

Comment:
Similar to my other comment, util code like this could be side-loaded via GCS

Reply:
Whoops, good catch. Thanks!